Defs/Ammo/Projectiles_Fragments.xml

```diff
@@ -14,7 +14,11 @@
 			<speed>60</speed>
 			<armorPenetrationSharp>4.5</armorPenetrationSharp>
 			<armorPenetrationBlunt>122.5</armorPenetrationBlunt>
-			<gravityFactor>5</gravityFactor>
+                        <lerpPosition>false</lerpPosition>
+                        <ballisticCoefficient>0.95~1.42</ballisticCoefficient>
+			<gravityFactor>1</gravityFactor>
+                        <mass>65~75</mass>
```

Comment:
Do we want to rename `mass` to something else, so that it's not going to potentially be mixed up with vanilla item `Mass`?